release_notes/ocp-4-15-release-notes.adoc

```diff
@@ -764,6 +764,13 @@ This release introduces a configuration adjustment in response to the transition
 
 To resolve these warnings, we recommend specifying `-1` or `auto` for the `maxConnections` field when tuning an IngressController. This choice allows HAProxy to dynamically calculate the maximum value based on the available resource limitations in the running container, which eliminates these warnings. (https://issues.redhat.com/browse/OCPBUGS-21803[*OCPBUGS-21803*])
 
+[id="ocp-4-15-auth-deployer-sa"]
+==== The deployer service account is no longer created if the DeploymentConfig cluster capability is disabled
+
+If you disable the `DeploymentConfig` cluster capability, the `deployer` service account its corresponding secrets are no longer created.
```

Comment:
```suggestion
If you disable the `DeploymentConfig` cluster capability, the corresponding secrets of the `deployer` service account are no longer created.
```

Reply:
Hope the technical meaning does not change?

Reply:
It does, but I can see why you changed it - I'm missing an "and" in there. Will update, thanks!